src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -116,7 +116,7 @@ class PublicationReleasePage extends Component<Props> {
             <Details summary="Download data files">
               <ul className="govuk-list govuk-list--bullet">
                 {data.dataFiles.map(({ extension, name, path, size }) => (
-                  <li key={path} data-testid="item-download">
+                  <li key={path}>
```

Comment:
do we not need the data-testid?

Reply:
I'm letting Mark add it back in when it's needed